app/src/features/react-hook-form-wrappers/FormattertTallTextField.tsx

```diff
@@ -1,47 +1,52 @@
 import { TextField, TextFieldProps } from "@navikt/ds-react";
-import { Controller, ControllerProps } from "react-hook-form";
+import { useController } from "react-hook-form";
+
+import { formatStrengTilTall } from "~/utils.ts";
 
 type FormattertTallTextFieldProps = TextFieldProps & {
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  control: any;
   name: string;
-  rules?: ControllerProps["rules"];
+  min?: number;
+  required?: boolean;
 };
 
 /** Et tekstfelt som formaterer innholdet til et tall med tusenvise mellomrom. */
 export const FormattertTallTextField = ({
-  defaultValue,
-  onChange: externalOnChange = () => {},
-  control,
   name,
-  rules,
+  min,
+  required,
   ...rest
 }: FormattertTallTextFieldProps) => {
-  return (
-    <Controller
-      control={control}
-      defaultValue={defaultValue}
-      name={name}
-      render={({ field }) => {
-        const { onChange, value, ref } = field;
+  const { field, fieldState } = useController({
+    name,
+    rules: {
+      required: required ? "Må oppgis" : false,
+      validate: (v) => {
```

Comment:
Tror det er lettere å gjøre valideringen mer manuell som dette. Problemet med å bruke rules.min direkte er at man da må si `valueAsNumber` også vil RHF selv gjør konverteringen til tall. Den vil da ikke ta høyde for bruk av komma, eller annet minustegn. Derfor best å gjøre det selv så vi kan påse vår egen Number transform